mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -478,6 +478,23 @@ func schemaNodeConfig() *schema.Schema {
 					ForceNew: true,
 					Description: `Setting this field will assign instances of this pool to run on the specified node group. This is useful for running workloads on sole tenant nodes.`,
 				},
+
+				"advanced_machine_features": &schema.Schema{
+					Type:        schema.TypeList,
+					Optional:    true,
+					MaxItems:    1,
+					Description: `Specifies options for controlling advanced machine features.`,
+					ForceNew:    true,
+					Elem:        &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"threads_per_core": {
+								Type:        schema.TypeInt,
+								Required:    true,
```

Comment:
We'll similarly need to mark this subfield with `ForceNew: true`! As the moment, we can create an invalid update by changing the value of the field (although adding/removing the block will be handled correctly).